sources/repo-insert

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+#     repo-scripts is a set of scripts to work with your repository
+#     Copyright (C) 2014 Evgeniy Alekseev
+#
+#     This program is free software; you can redistribute it and/or modify
+#     it under the terms of the GNU General Public License as published by
+#     the Free Software Foundation; either version 3 of the License, or
+#     (at your option) any later version.
+#
+#     This program is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#     GNU General Public License for more details.
+#
+#     You should have received a copy of the GNU General Public License
+#     along with this program; if not, see http://www.gnu.org/licenses
+#     or write to the Free Software Foundation,Inc., 51 Franklin Street,
+#     Fifth Floor, Boston, MA 02110-1301  USA
+
+
+# functions
+error_mes() {
+  if [ "${USECOLOR}" == "yes" ]; then
+    local _red='\e[0;31m'
+    local _bwhite='\e[1;37m'
+    local _cclose='\e[0m'
+  fi
+  case "${1}" in
+    "config"    ) echo "${_red}[EE] ${_bwhite}Configuration file is not set${_cclose}" ;;
+    "file"      ) echo "${_red}[EE] ${_bwhite}'${2}' is a file${_cclose}"              ;;
+    "unknown"   ) echo "${_red}[EE] ${_bwhite}Unknown error${_cclose}"                 ;;
+    "flag"      ) echo "${_red}[EE] ${_bwhite}Unknown flag${_cclose}"                  ;;
+  esac
+  exit 1
+}
+
+func_help() {
+  echo -e "Script written on BASH to add a package in the repository"
+  echo -e "Usage: repo-insert [ -c /etc/repo-scripts.conf ] [ -v | --version ] [ -h | --help ] PACKAGE1 PACKAGE2 ..."
```

Comment:
it would be great if you will update help message according to the available flags